spec/controllers/api/teams_controller_spec.rb

```diff
@@ -574,6 +648,28 @@
 
   private
 
+  def prepare_transferred_encryptable
```

Comment:
maybe use a fabricator?

Reply:
I moved the method to test helper, because we use it in encrpytables controller spec